awscli/examples/cloudformation/deploy.rst

```diff
@@ -1,6 +1,9 @@
 Following command deploys template named ``template.json`` to a stack named
 ``my-new-stack``::
 
-
     aws cloudformation deploy --template-file /path_to_template/template.json --stack-name my-new-stack --parameter-overrides Key1=Value1 Key2=Value2 --tags Key1=Value1 Key2=Value2
 
```

Comment:
Please delete line 6 - 9 here, as we dont want a nearly duplicated example for the usage of an optional flag. Customer can refer to previous section for the usage of the `disable-rollback` flag, similar for other optional flags.